  0%|                                                                                                | 0/150000 [00:00<?, ?it/s]Traceback (most recent call last):
  File "/Users/mingu/Desktop/CODING/brAIn/2th/brAIn_2th_Torch_Session/week3/train.py", line 76, in <module>
    main(args)
  File "/Users/mingu/Desktop/CODING/brAIn/2th/brAIn_2th_Torch_Session/week3/train.py", line 70, in main
    trainer.train()
  File "/Users/mingu/miniforge3/envs/ag/lib/python3.10/site-packages/transformers/trainer.py", line 1780, in train
    return inner_training_loop(
  File "/Users/mingu/miniforge3/envs/ag/lib/python3.10/site-packages/transformers/trainer.py", line 2118, in _inner_training_loop
    tr_loss_step = self.training_step(model, inputs)
  File "/Users/mingu/miniforge3/envs/ag/lib/python3.10/site-packages/transformers/trainer.py", line 3036, in training_step
    loss = self.compute_loss(model, inputs)
  File "/Users/mingu/Desktop/CODING/brAIn/2th/brAIn_2th_Torch_Session/week3/src/trainer.py", line 20, in compute_loss
    output = model(image)
  File "/Users/mingu/miniforge3/envs/ag/lib/python3.10/site-packages/torch/nn/modules/module.py", line 1501, in _call_impl
    return forward_call(*args, **kwargs)
  File "/Users/mingu/Desktop/CODING/brAIn/2th/brAIn_2th_Torch_Session/week3/src/model/cnn.py", line 13, in forward
    x = self.conv1(x)
  File "/Users/mingu/miniforge3/envs/ag/lib/python3.10/site-packages/torch/nn/modules/module.py", line 1501, in _call_impl
    return forward_call(*args, **kwargs)
  File "/Users/mingu/miniforge3/envs/ag/lib/python3.10/site-packages/torch/nn/modules/conv.py", line 463, in forward
    return self._conv_forward(input, self.weight, self.bias)
  File "/Users/mingu/miniforge3/envs/ag/lib/python3.10/site-packages/torch/nn/modules/conv.py", line 459, in _conv_forward
    return F.conv2d(input, weight, bias, self.stride,
RuntimeError: Given groups=1, weight of size [16, 128, 3, 3], expected input[1, 1, 28, 28] to have 128 channels, but got 1 channels instead
